packages/cli/src/utils/generate.ts

```diff
@@ -60,6 +60,31 @@ function filterAndCopyPackageJson(basePath: string) {
   })
 }
 
+// Temporary array of strict rules enabled so far.
+const TS_CONFIG_STRICT_RULES_ENABLED = ['noImplicitAny'] as const
+
+/**
+ * Modify TypeScript compilation settings (tsconfig.json) to disable specific strict
+ * type checking rules when files are moved to the .faststore folder. 
+ * The idea is to change the strict to false when all strict rules are migrated.
```

Comment:
If I got it right, we will remove this part when all rules are migrated, right?
**suggestion**: maybe we can add a todo markup here - // TODO: